### ISSUE TITLE
Explore REST fallback...

### ISSUE THREAD

**Issue by AnashOommen** [googleads/google-ads-dotnet#508]
`Grpc.Core` is now sunset, and `Grpc.Net.Client` doesn't do a good job of working well on .NET Framework. Yet, we have a lot of users on .NET Framework who cannot easily move over to .NET 5 / .NET Core. We saw this issue come up in several recent bugs:

* https://github.com/googleads/google-ads-dotnet/issues/503
* https://github.com/googleads/google-ads-dotnet/issues/501
* https://github.com/googleads/google-ads-dotnet/issues/488
* https://github.com/googleads/google-ads-dotnet/issues/482
* https://github.com/googleads/google-ads-dotnet/issues/480

Let's explore using REST fallback as an option for these platforms.

**Comment by AnashOommen** [comment]
Hi @Yao725 no, there wasn't anyone who asked for REST transport for the .NET library so far. Are you facing an issue?

**Comment by Yao725** [comment]
Yeah, I‘m exploring the use of .NET Framework with Grpc.Net.Client to call the Ads API, but the minimum OS version requirement blocked me, so I'm interested in the progress of this issue.
